CONTRIBUTING.md

```diff
@@ -83,35 +83,28 @@ This section describes in some detail how changes can be made and proposed with
    See <!-- TODO branch section link --> for when a different branch should be used instead.
    Make sure to go through the [pull request template](#pull-request-template) in the pre-filled default description.
 
-### Making patches {#submitting-changes-making-patches}
-
-- If you removed pkgs or made some major NixOS changes, write about it in the release notes for the next stable release. For example `nixos/doc/manual/release-notes/rl-2003.xml`.
-
-- Test your changes. If you work with
-
-  - nixpkgs:
-
-    - update pkg
-      - `nix-env -iA pkg-attribute-name -f <path to your local nixpkgs folder>`
-    - add pkg
-      - Make sure it’s in `pkgs/top-level/all-packages.nix`
-      - `nix-env -iA pkg-attribute-name -f <path to your local nixpkgs folder>`
-    - _If you don’t want to install pkg in you profile_.
-      - `nix-build -A pkg-attribute-name <path to your local nixpkgs folder>` and check results in the folder `result`. It will appear in the same directory where you did `nix-build`.
-    - If you installed your package with `nix-env`, you can run `nix-env -e pkg-name` where `pkg-name` is as reported by `nix-env -q` to uninstall it from your system.
-
-  - NixOS and its modules:
-    - You can add new module to your NixOS configuration file (usually it’s `/etc/nixos/configuration.nix`). And do `sudo nixos-rebuild test -I nixpkgs=<path to your local nixpkgs folder> --fast`.
-
-
-- [Rebase](https://git-scm.com/book/en/v2/Git-Branching-Rebasing) your branch against current `master`.
+6. Respond to review comments, potential CI failures and potential merge conflicts by updating the pull request.
+   Always keep the pull request in a mergeable state.
 
+   To add new commits, repeat steps 2-3 and push the result using
+   ```
+   git push
+   ```
 
-### (Creating a pull request)
+   To change existing commits you will have to [rewrite Git history](https://git-scm.com/book/en/v2/Git-Tools-Rewriting-History).
+   Useful Git commands that can help a lot with this are `git commit --patch --amend` and `git rebase --interactive`.
+   With a rewritten history you need to force-push the commits using
+   ```
+   git push --force-with-lease
+   ```
 
-When pull requests are made, our tooling automation bot,
-[OfBorg](https://github.com/NixOS/ofborg) will perform various checks
-to help ensure expression quality.
```

Comment:
That is a good point, this is one sentence I removed because I didn't quiet know where it would fit, but I think this should work: 182d3551c5e409ee43a57e65a1160e2cf37a4b4e